MAINTAINERS.md

```diff
@@ -0,0 +1,8 @@
+## Maintainers
+
+| Maintainer | GitHub ID | Affiliation |
+| --------------- | --------- | ----------- |
+| Placeholder | [placeholder](https://github.com/userid) | Company |
```

Comment:
Currently, we have @idanl21 , @ido-opster , and @dbason as maintainers. They can get added in separate PRs.

Reply:
**I will add them in shortly for you**